app/views/questions/show.html.haml

```diff
@@ -12,13 +12,13 @@
           %p.typography--heavy
             = link_to(@asking_person.display_name, person_questions_written_path(@asking_person.graph_id))
         - if @question.asked_at_date
-          %time{ datetime: @question.asked_at_date.strftime('%d-%m-%Y') }= "#{t('.asked_on').capitalize} #{@question.asked_at_date.strftime('%d %B %Y')}"
+          %time{ datetime: l(@question.asked_at_date, format: '%Y-%m-%d') }= "#{t('.asked_on').capitalize} #{l(@question.asked_at_date, format: :default)}"
```

Comment:
Should the hard-coded format not live in the translation file if we are using it in multiple places

Reply:
We should avoid hard-coding like this at all costs

Reply:
Okay, I'll put this into the translation.